install-frontend.js

```diff
@@ -14,10 +15,18 @@ const execAndPrint = (command) => print(execSync(command))
 
 const installFrontendScript = () => {
   printInfo('Removing previous frontend repo')
-  execAndPrint('rm -rf ./Bears-Team-6')
```

Comment:
I think you write a function to process these if/else, to prevent duplicate code